packages/profile/src/components/inventory/token/token.tsx

```diff
@@ -30,10 +30,16 @@ import { formatEther } from "viem";
 import { useAccount } from "@/hooks/account";
 import { useToken } from "@/hooks/token";
 import { TokenPair } from "@cartridge/utils/api/cartridge";
+import { usePostHog } from "posthog-js/react";
+import { useEffect } from "react";
 
 export function Token() {
   const { address } = useParams<{ address: string }>();
   const location = useLocation();
+  const posthog = usePostHog();
+  useEffect(() => {
+    posthog.setPersonPropertiesForFlags({ address });
```

Comment:
can we move this to the connection where we set other posthog metadata?

Reply:
moved it to `useAccount` hook